docs/docs/project_configuration/upgrading.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 9
+description: Learn how to upgrade your project to incorporate improvements in the toolkit.
+---
+
+# Upgrading your project
+
+If you have generated a project using an older version of the Flutter News Template, you can upgrade your project to take advantage of any fixes and improvements.
+
+:::caution
+It's recommended that you use a version control tool like `git` and that you have committed all changes before trying to upgrade. Please make sure you have a backup of your project before proceeding so that you can revert the changes if you encounter any issues during the upgrade process.
+:::
+
+## Upgrade the template
+
+In order to upgrade an existing project, you must first upgrade to the latest available version of the [`flutter_news_template`](https://brickhub.dev/bricks/flutter_news_template).
+
+:::info
+You can check the local version of the `flutter_news_template` by running `mason list --global`
+
+```
+mason list --global
+/Users/me/.mason-cache/global
+└── flutter_news_template 1.0.0 -> registry.brickhub.dev
+```
+
+:::
+
+If you have an outdated version installed, upgrade by running `mason upgrade --global`
+
+:::note
+Running `mason upgrade --global` will also upgrade other globally installed templates. If you wish to avoid this you can re-install just the `flutter_news_template` via:
```

Comment:
```suggestion
Running `mason upgrade --global` will also upgrade other globally installed templates. If you wish to avoid this, you can re-install just the `flutter_news_template` via:
```